src/rt/sched/cown.h

```diff
@@ -901,6 +881,31 @@ namespace verona::rt
       }
     }
 
+    /**
+     * Ensures that any muted recipients will become unmutable if any of the
+     * message cowns are overloaded.
+     */
+    static inline void backpressure_ensure_progress(MessageBody* body)
+    {
+      bool requires_unmute = std::any_of(
+        &body->cowns[0], &body->cowns[body->count], [](const auto* c) {
+          return c->backpressure.load(std::memory_order_acquire).overloaded();
+        });
+      yield();
+      if (
+        !requires_unmute
+#ifdef USE_SYSTEMATIC_TESTING
+        && !Systematic::coin(3)
+#endif
+      )
+        return;
+
+      for (size_t i = body->index; i < body->count; i++)
```

Comment:
nit: range loop?

Reply:
`body->cowns` is a raw pointer. A container should be added eventually, like vector but with snmalloc.